py/pycompiler.vim

```diff
@@ -747,6 +746,8 @@ function s:PythonCompiler.compile_identifier(node)
     let name = 'vval'
   elseif name =~ '^[sa]:'
     let name = name[2:]
+  elseif index(s:reserved_keywords, name) >= 0
```

Comment:
and I think we should use `dict` instead of `list`?

sererved_keywords are small, but this line will be called frequently. O(1) is better I guess.

[optional] Or/And you can list up reserved name only vimlparser.vim uses.